examples/person/valid/tb.json

```diff
@@ -1,5 +1,5 @@
 {
-  "@context": "https://schema.org/",
+  "@context": "https://schema.org",
```

Comment:
Müssen wir bei einer Formatierungsrunde bei allen einmal anpacken.

Reply:
Ja. Die bessere Form ist mit / am Ende. Es ist sogar so: wenn der Slash fehlt, wird er erst noch zugefügt vom Webserver oder Browser, weil sonst die Anfrage invalide ist. Siehe RFC 2616. Letztlich wird das, wie gesagt, von server oder client gefixed. Aber "
It allows for faster loading time by telling the browser to look for the default file. "

Reply:
https://schema.org/docs/actions.html macht es selber aber falsch.
Es gibt halt ein 301 mehr als nötig. Curlt mal selber: http://schema.org .